lib/fog/kubevirt/compute/compute.rb

```diff
@@ -269,6 +126,7 @@ def deep_merge!(source_hash, other_hash, &block)
       class Real
         require "ostruct"
         include Shared
+        include Utils
```

Comment:
We can skip the inclusion if we'll refer the ExceptionWrapper as: `Kubevirt::Utils::ExceptionWrapper.new(client, version)`

Reply:
this seems to be less code. It will only contain this class due to `require` I added.